linker/shared/src/main/scala/org/scalajs/linker/analyzer/Infos.scala

```diff
@@ -281,6 +284,20 @@ object Infos {
       this
     }
 
+    def addReferencedClass(tpe: TypeRef): this.type =
+      addReferencedClass(baseNameOf(tpe))
+
+    def addReferencedClass(cls: String): this.type = {
+      referencedClasses += cls
+      this
+    }
+
+    def maybeAddReferencedClass(tpe: Type): this.type = tpe match {
+      case ClassType(cls) => addReferencedClass(cls)
+      case ArrayType(ref) => addReferencedClass(ref)
```

Comment:
We usually use `typeRef` instead of `ref` as the name of a val that represents a `TypeRef`.